src/commands/node.mjs

```diff
@@ -1002,6 +1227,30 @@ export class NodeCommand extends BaseCommand {
               })
             }
           })
+          .command({
+            command: 'refresh',
+            desc: 'Refresh a node',
```

Comment:
`Refresh a node` --> `Reset and restart a node`

Reply:
Since you are out, I'll merge this and create a small PR to fix it.